tests/integration/copy-address.spec.ts

```diff
@@ -0,0 +1,45 @@
+import { createTestSelector, BrowserDriver, setupBrowser } from './utils';
+import { WalletPage } from './page-objects/wallet.page';
+import { ScreenPaths } from '@common/types';
+import { validateStacksAddress } from '@common/stacks-utils';
+import { UserAreaSelectors } from './user-area.selectors';
+
+
+jest.setTimeout(30_000);
+jest.retryTimes(process.env.CI ? 2 : 0);
+describe(`Copy Address`, () => {
+  let browser: BrowserDriver;
+  let wallet: WalletPage;
+
+  const isCopied = async () => {
+    console.log('is copied?')
+    let copied = await wallet.page.evaluate(() => window.navigator.clipboard.readText());
+    // Ex: SP2BK439HDX54D2Z8G642VFY9KHZJ3P6Z9NEJWDHW
+    expect(validateStacksAddress(copied)).toBe(true)
+  }
+
+  beforeEach(async () => {
+    browser = await setupBrowser();
+    wallet = await WalletPage.init(browser, ScreenPaths.INSTALLED);
+  }, 10000);
+
+  afterEach(async () => {
+    try {
+      await browser.context.close();
+    } catch (error) {}
+  });
+
+  it('should be able to copy address', async () => {
+    await wallet.signUp();
+    await wallet.page.click(createTestSelector(UserAreaSelectors.AccountBalancesCopyAddress));
+    await isCopied()
```

Comment:
This is really just a preference, but I find it odd not to have the `expect` clause in the `it` body.

Could `isCopied()` return `wallet.page.evaluate(() => window.navigator.clipboard.readText())`

and the `expect` be inline, even if repeated, `expect(await isCopied()).toBeTruthy()`?

Reply:
Sure, this will be more readable.